src/core/shape/vertex.js

```diff
@@ -896,6 +899,56 @@ p5.prototype.quadraticVertex = function() {
  * @alt
  * 8 points making 4 lines
  *
+ * @example
+ * <div>
+ * <code>
+ * // Shape with a hollow section in the middle. Click to change the number of sides.
+ * // Custom shapes drawn with beginShape() would make use of the tessellator by default.
+ * // Custom shapes with different z-values (i.e non-coplanar shapes) would not use the tessellator.
+ * // So, it is recommended to draw all custom shapes on a flat z-plane.
+ *
+ * let sides = 3;
+ * let angle, px, py;
+ *
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ *   setAttributes('antialias', true);
+ *   fill(237, 34, 93);
+ *   strokeWeight(3);
+ * }
+ *
+ * function ngon(n, x, y, d) {
```

Comment:
Can we move this function to the end? I think it is more clear what it does if one sees `draw()` first.